### PR TITLE
Create new file of places to eat in the 'Burgh

### LINKED ISSUE #4: GithubTeacher favorite places to eat
Let's add a document using:
- [x] Create a branch
- [x] Add a new file
- [x] Commit my file
- [x] Open Pull Request
- [ ] Discuss
- [ ] Merge

@carolynshin do you have any specific recommendations?

### LINKED ISSUE #6: kaseybyrne places to eat
- [ ] create a branch
- [ ] add a new file
- [ ] commit my file
- [ ] open pull request
- [ ] discuss
- [ ]  merge

### PR DESCRIPTION
**Pull request by kaseybyrne**
Resolves #6

### CODE REVIEW

File: eats/pittsburgh-pa.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+### Pittburgh, PA 
+
+* Primanti Brothers 
+* Wholey's Fish Market
```
Comment (kaseybyrne) [inline eats/pittsburgh-pa.md:4]: both places are cash only!

**Comment by cowperthwait** [comment]
see e.g. [worst restaurant in pittsburgh - Google Search](https://www.google.com/search?client=safari&rls=en&q=worst+restaurant+in+pittsburgh&ie=UTF-8&oe=UTF-8)

**Comment by watridge** [comment]
Nice food!

**Comment by cowperthwait** [comment]
lgtm; 🐑

**Pull request merged**